apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -240,7 +242,7 @@ export class DetailViewContents extends React.Component {
     if (this.props.canLock) {
       // Render the select with the lock button in a fancy InputGroup
       return (
-        <InputGroup style={{maxWidth: 200, marginRight: 5}}>
+        <InputGroup style={{maxWidth: 200, marginRight: 5, marginLeft: 5}}>
```

Comment:
according to our [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react) these inline styles should be in the `styles` object